internal/util/cmdutil/cmdutil.go

```diff
@@ -82,3 +84,22 @@ func ResolveAbsAndRelPaths(path string) (string, string, error) {
 
 	return relPath, absPath, nil
 }
+
+// DockerCmdAvailable runs `docker ps` to check that the docker command is
+// available, and returns an error with installation instructions if it is not
+func DockerCmdAvailable() error {
+	cmd := exec.Command("docker", "ps")
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	if err != nil {
+		if strings.Contains(stderr.String(), "Cannot connect to the Docker daemon") {
+			return fmt.Errorf(stderr.String())
+		}
+		return fmt.Errorf("docker is required to run this command\n" +
+			"To install docker, please follow the instructions on the docker installation page.\n" +
+			"Docker installation instructions can be found here:\n" +
+			"https://docs.docker.com/get-docker/")
```

Comment:
tip: you can use Go's string literal with backtick to make it easier craft the error message.

Also, we can shorten the error message saying:
`docker is required to run this command.
To install docker, follow the instructions at https://docs.docker.com/get-docker/
`

Reply:
done